x-pack/plugins/cases/server/connectors/cases/schema.ts

```diff
@@ -25,13 +25,7 @@ const GroupingSchema = schema.arrayOf(schema.string(), { minSize: 0, maxSize: 1
 const RuleSchema = schema.object({
   id: schema.string(),
   name: schema.string(),
-  /**
-   * TODO: Verify limits
-   */
-  tags: schema.arrayOf(schema.string({ minLength: 1, maxLength: 50 }), {
```

Comment:
Rules do not impose any limits on tags (https://github.com/elastic/kibana/blob/80640cf1e5aeb967d53dc4cdaa03a71c4552a0d4/x-pack/plugins/alerting/common/routes/rule/apis/create/schemas/v1.ts)